src/app/core/services/observation/observation.service.ts

```diff
@@ -17,6 +17,7 @@ import { LoggingService } from '../../../modules/shared/services/logging/logging
 import { DbHelperService } from '../db-helper/db-helper.service';
 import { LogLevel } from '../../../modules/shared/services/logging/log-level.model';
 import { NSqlFullUpdateObservable } from '../../helpers/nano-sql/NSqlFullUpdateObservable';
+import { DraftToRegistrationService } from '../draft/draft-to-registration.service';
 
 const DEBUG_TAG = 'ObservationService';
 
```

Comment:
Hvis vi skal beholde denne, synes jeg vi bør dokumentere hva denne tjenesten har ansvar for